11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,39 @@
+.container{
+  display: flex;
+  flex-direction: column;
+
+}
+.box1{
+  background: #000 url("../img/bg2.jpg") no-repeat right;
+  background-size: cover;
+  height: 400px;
+  color: white;
+  font-size: 1em;
+}
+.box2{
+  background-color: #666;
+  color: white;
+  font-size: 8px;
+}
+.box3{
+  background-color: #666;
+  color: white;
+  font-size: 8px;
+}
+@media screen and (min-width: 320px) {
+  .container::before {
+    content: "320px";
+    color: orange;
+  }
+}
+@media screen and (min-width: 480px) {
+  .container::before {
+    content: "480px";
+    color: maroon;
+  }
+}
+@media screen and (min-width: 620px) {
+  .container::before {
+    content: "620px";
+    color: orange;
+  }
```

Comment:
Also, the two text sections (box2 and box3) are supposed to transition to being side-by-side.